app/src/main/java/com/github/polypoly/app/ui/map/MapInitializationHelpers.kt

```diff
@@ -77,33 +81,78 @@ fun initMapView(context: Context): MapView {
  * @param gameViewModel GameViewModel to use for game business logic. Null if no game is going on.
  * @return the marker that was added
  */
-fun addMarkerTo(mapView: MapView, location: LocationProperty, zoneColor: Int,
-                mapViewModel: MapViewModel, gameViewModel: GameViewModel?
+fun addMarkerTo(
```

Comment:
I find it cleaner to put one parameter per line for readability

Reply:
done